bulbs/poll/api.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.routers import DefaultRouter
+
+from .viewsets import PollViewSet, AnswerViewSet
+
+router = DefaultRouter(trailing_slash=True)
+
+api_v1_router = DefaultRouter()
+api_v1_router.register(
```

Comment:
Yeah we've been using flake8 with tunic/combine.

It's a good first-step linter that doest stuff we can all agree on. I recommend rolling this out to other projects.